File: BackUpCode/MakingApps-master/MakingApps-master/AndroidStudioProjects/viewpager22/app/src/main/java/com/example/user/viewpager2/ViewPagerFragment.java
package com.example.user.viewpager2;

import android.app.Fragment;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

public class ViewPagerFragment extends Fragment {
    public static ViewPagerFragment newInstance(int image) {

        Bundle args = new Bundle();
        args.putInt("image", image);

        ViewPagerFragment fragment = new ViewPagerFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.viewpager_layout, container, false);

        ImageView imageView = (view).findViewById(R.id.viewpager_imageview);
        imageView.setImageResource(getArguments().getInt("image"));

        return view;
    }
}

File: KotlinViewPagerTest_20190131/app/src/main/java/com/updatetest/kotlinviewpagertest_20190131/ViewPagerFragment.kt
package com.updatetest.kotlinviewpagertest_20190131

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView

class ViewPagerFragment : Fragment() {
    fun loadImage(image : Int?) : Fragment {
        var args : Bundle? = Bundle()
        args?.putInt("image", image!!)

        var fragment : ViewPagerFragment? = ViewPagerFragment()
        fragment?.arguments = args
        return fragment!!
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        var view : View = inflater.inflate(R.layout.viewpagerfragment, container, false)

        var imageView : ImageView? = view.findViewById(R.id.viewpager_imageview)
        imageView?.setImageResource(arguments!!.getInt("image"))
        return view
    }
}